h2o-core/src/main/java/hex/PartialDependence.java

```diff
@@ -128,9 +129,11 @@ public void compute2() {
                 if (_model_id.get()._output.nclasses() == 2) {
                   meanResponse[which] = preds.vec(2).mean();
                   stddevResponse[which] = preds.vec(2).sigma();
+                  stdErrorofTheMeanResponse[which] = preds.vec(2).sigma()/ preds.numRows();
```

Comment:
Shouldn't it be sqrt? SEM = SD / sqrt(n)

Reply:
whoops. i thought i changed that.